internal/app/go-filecoin/paymentchannel/testing/fake_paych_actor_iface.go

```diff
@@ -0,0 +1,100 @@
+package testing
+
+import (
+	"context"
+	"testing"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/builtin/paych"
+	"github.com/filecoin-project/specs-actors/support/mock"
+	spect "github.com/filecoin-project/specs-actors/support/testing"
+	"github.com/ipfs/go-cid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/internal/pkg/block"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/crypto"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/vm"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/gas"
+)
+
+type FakePaychActorIface struct {
```

Comment:
It's confusing for a concrete implementation to end with iface. `FakePaychActorUtil`? `FakePaychActorConnector`? `FakePaychActorServices`?